src/compiler/crystal/syntax/lexer.cr

```diff
@@ -491,21 +491,21 @@ module Crystal
             when '\\'
               case char = next_char
               when 'a'
-                io << "\u{7}"
+                io << '\u{7}'
```

Comment:
Or shouldn't this be rather `io << '\a'`?

Reply:
yes